README.md

```diff
@@ -208,13 +221,52 @@ const stream = customer.reportStream({
 // Rows are streamed in one by one
 for await (const row of stream) {
     // Break the loop to stop streaming
-    if(someLogic) {
+    if (someLogic) {
         break
     }
 }
 ```
 <!-- prettier-ignore-end -->
 
+---
+
+## Retrieve Keywords with a raw stream
+
+Returns the raw stream so that events can be handled manually. This is the recommended method of retrieving data with over 10,000 rows. For more information on Google's stream methods please [consult their docs](https://github.com/googleapis/gax-nodejs/blob/master/client-libraries.md#server-streaming).
```

Comment:
Nice work updating the docs to reflect the changes! However, is this 10,000 rows recommendation still true?